src/backend/src/routes/projects.routes.ts

```diff
@@ -36,7 +35,6 @@ projectRouter.post(
   body('otherConstraints').isArray(),
   intMinZero(body('otherConstraints.*.id').optional()),
   nonEmptyString(body('otherConstraints.*.detail')),
-  body('wbsElementStatus').custom((value) => Object.values(WbsElementStatus).includes(value)),
```

Comment:
we should also take it out of the create project endpoint/route too im realizing

Reply:
oh I just checked the code for CREATE project endpoint in its controller and routes file, but its route didn't require any wbsElementStatus as input and in the project controllers, it only created the summary (string) field for the new project. 
I guess it will be fine then?

Reply:
you're right, didn't realize that. nice!